### PR TITLE
update basic page

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: web/themes/custom/pineapple/templates/content/node--page--full.html.twig [hunk 1]
```diff
@@ -80,12 +80,19 @@
     view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
   ]
 %}
+{%
+  set title_classes = [
+  'page-title',
+  content.field_bp_hide_page_heading.0['#markup'] == 'Hide' ?
+    'visually-hidden',
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/templates/content/node--page--full.html.twig]: Put 'visually-hidden' on line 86 and also replace ".0" with `|field_value|first` it's better practice

Comment (katlazarova) [reply web/themes/custom/pineapple/templates/content/node--page--full.html.twig]: done

File: web/themes/custom/pineapple/composer.json [hunk 2]
```diff
@@ -32,6 +32,11 @@
             "libraries/{$name}": [
                 "type:npm-asset"
             ]
+        },
+        "patches": {
+            "drupal/paragraphs": {
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/composer.json]: Add this patch above the existing one 
```
"drupal/core": {
                "Media library widget + Paragraphs broken": "https://www.drupal.org/files/issues/2019-05-16/drupal-media-library-paragraphs-3003150-38-8.x.patch",
            },
```

Comment (ElXyfer) [reply web/themes/custom/pineapple/composer.json]: Run composer install to make sure it's aded

Comment (katlazarova) [reply web/themes/custom/pineapple/composer.json]: done

**Pull request merged**